src/shell/lisa_shell

```diff
@@ -308,19 +319,35 @@ nosetests -v --nocapture --nologcapture \
 }
 
 function lisa-test {
-CMD=${1:-help}
-echo
-case "x${CMD^^}" in
-"xHELP")
-	_lisa-test-usage
-	;;
-*)
-	_lisa-test $*
-        local retcode=$?
-esac
-echo
+local iterations=0 # Means use default - see libs/utils/test.py
+local have_more_args=no
+
 echo
-return $retcode
+while [[ $# -gt 0 ]]; do
+        case "$1" in
+                help|-h|--help)
+                        _lisa-test-usage
+                        return 0
+                        ;;
+                -n|--iterations)
+                        iterations="$2"
+                        shift
+                        ;;
+                *)
```

Comment:
Perhaps I'm note really getting what you wanna do here.
Isn't this extremely fragile?

AFAIU we are forced to put `-n/--interations `before the remaining params, otherwise we will pass it to the `__list_test` instead and run just 1 iteration.

Did not tried but I still have the impression that getopts can be useful to make this more robust... maybe I'll give a try later. Can you just post here an example of how you wanna call a test with parameters and multiple iterations?